sui/its.js

```diff
@@ -228,15 +229,35 @@ async function registerCustomCoin(keypair, client, config, contracts, args, opti
     const deployConfig = { client, keypair, options, walletAddress };
     const [symbol, name, decimals] = args;
 
+    const unvalidatedParams = { isNonEmptyString: { symbol, name } };
+
     if (options.salt) {
-        validateParameters({
-            isHexString: { salt: options.salt },
-        });
+        unvalidatedParams.isHexString = { salt: options.salt };
     }
 
+    validateParameters(unvalidatedParams);
+
     // Deploy token on Sui
     const [metadata, packageId, tokenType, treasuryCap] = await deployTokenFromInfo(deployConfig, symbol, name, decimals);
 
+    // Mint pre-registration coins
+    const amount = !isNaN(options.mintAmount) ? parseInt(options.mintAmount) : 0;
+    if (amount) {
+        const unitAmount = getUnitAmount(options.mintAmount, decimals);
+
+        const mintTxBuilder = new TxBuilder(client);
+
+        const coin = await mintTxBuilder.moveCall({
+            target: `${SUI_PACKAGE_ID}::coin::mint`,
```

Comment:
when you split cmds, mint coin should be a separate generic util defined in `sui/tokens.js` instead

Reply:
we have 2 separate generic commands for minting coins, however the version created by @benjamin852 should be moved from `its.js` to `tokens.js`